src/main/java/controller/CreateMessage.java

```diff
@@ -91,24 +94,14 @@ public void createMessageWithoutChatIdPathVariable(HttpServletRequest request) {
      * Returns MessageId of the sent Message.
      */
     @PostMapping("/users/{userId}/chats/{chatId}/messages")
-    public Map<String, Object> createMessage(@PathVariable("userId") String userIdString, @PathVariable("chatId") String chatIdString, @RequestBody Map<String, String> requestBody, HttpServletRequest request) {
-        
+    public Map<String, Object> createMessage(@PathVariable("userId") String userIdString, @PathVariable("chatId") String chatIdString, @RequestParam(value = "textContent", required = false) String textContent, @RequestParam(value = "file", required = false) MultipartFile file, HttpServletRequest request) throws IOException {
```

Comment:
This line is too long.
Please break this into multiple lines.

Reply:
Done